automation/Clean.ps1

```diff
@@ -0,0 +1,51 @@
+
+param(
+    [string]$SwitchName = "Default Switch",
+
+    [Parameter(Mandatory=$true)]
+    [string]$VMName,
+
+    [Parameter()]
+    [string]$Platform = "Hyper-V"
+)
+
+if ($VMName) {
+    try{
+        if ($Platform -eq "Hyper-V") {
+            $dirPath = "C:\Users\$env:USERNAME\.minikube\machines\$VMName"
```

Comment:
Split HyperV and Virtual box to seperate functions to make it easier to mange each of them

Reply:
yup, eventually will do, this is in draft state, not finished